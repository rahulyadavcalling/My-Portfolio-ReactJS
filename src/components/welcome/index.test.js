import React from "react";
import renderer from "react-test-renderer";

import Welcome from "./index.js";

it("renders correctly when there are no items", () => {
  const tree = renderer.create(<Welcome />).toJSON();
  expect(tree).toMatchSnapshot();
});
