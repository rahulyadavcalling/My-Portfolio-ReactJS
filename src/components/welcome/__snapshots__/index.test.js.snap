// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="sc-bdVaJa dsPhUh"
>
  <h4
    className="sc-bwzfXH jFIetV"
  >
    Hello World;
    <br />
    <br />
    I am
  </h4>
  <h1
    className="sc-htpNat frHDCV"
  >
    <b>
      Reyansh
      <br />
       Mishra
    </b>
  </h1>
  <br />
  <h4
    className="sc-bxivhb dngKrB"
  >
    Mobile / Web Developer
  </h4>
</div>
`;
